Standards/scs-0112-v1-volume-type-standard.md

```diff
@@ -0,0 +1,227 @@
++--------------------+--------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| access_project_ids | None                                                                                                                                                         |
+| description        | Content will be replicated three times on spinning disks to ensure consistency and availability for your data. The volumes can be used in AZ 1,3,4.          |
+| id                 | d63307fb-167a-4aa0-9066-66595ea9fb21                                                                                                                         |
+| is_public          | True                                                                                                                                                         |
+| name               | hdd-three-replicas-AZ134                                                                                                                                     |
+| properties         |                                                                                                                                                              |
+| qos_specs_id       | None                                                                                                                                                         |
++--------------------+--------------------------------------------------------------------------------------------------------------------------------------------------------------+
+```
+
+### Multiattach
+
+A few backends enable providers to allow volumes to be attached to mulitple VMs simultaneously. The OpenStack API can be used to enable the usage of multiattach through the volume type. The property is also automatically shown to users. If a deployer wants to use this feature they have to do it in this way. So users will always see whether a volume type can be used for mulitattach volumes or not. Nevertheless due to other problems, that might occur when using the multiattach feature such as the requirement of special file systems and suitable backends, the SCS standard will only view this as OPTIONAL.
+
+### Volume QoS
+
+Quality of Service parameters for volumes can be defined in a volume qos object. While admins can use these qos objects directly on volumes, a user is not able to see such an object. Instead there can be ONE volume qos object associated to a volume type, which is then be used on all volumes created from that volume type. Through this indirection, volume types also working without these associated qos objects and this feature not being heavily used, the SCS standard will currently only view this as an OPTIONAL volume type feature.
```

Comment:
>  While admins can use these qos objects directly on volumes

I do not understand this sentences. Can an admin define QoS objects for volumes directly? If so, how does the underlying infrastructure reflect this QoS? To reduce potential of confusion, I would remove this sentence and focus on user user only.


  ```suggestion
Quality of Service parameters for volumes can be defined in a volume QoSs object by admins. A user is not able to see such QoS objects. Instead there can be ONE volume QoS object associated to a volume type only, which is then be used on all volumes created from that volume type. Through this indirection, volume types also working without these associated QoS objects and this feature not being heavily used, the SCS standard will currently only view this as an OPTIONAL volume type feature.
```

Reply:
Okay so this seem to need further explanation:

`volume qos` is an extra ressource (or object) in openstack. Administrators can use it on volumes directly. Normal user ar not able to see these objects. So to use `volume qos` objects or QoS for volumes in general , an administrator has to create a `volume qos` object and associate it with a volume type (only one qos object per type). It is then applied to every volume created with this volume type. BUT still only admins are able to see the associated `volume qos` id in the volume type.

I think I will rephrase the fist sentences.

Reply:
optional aspects were removed from this standard - so this is no longer part of it.